README.md

```diff
@@ -204,7 +205,7 @@ The UI class contains static methods for displaying the list of books in the HTM
 
 Together, these classes allow the user to add books to a list, remove books from the list, and have the list persist even after the page is refreshed.
   
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>..
```

Comment:
- [ Required ] I think it would be better if you fix the `License` link on your readme as suggested by the previous reviewer. Currently, the license link leads to a `404` page not found error. It would be better if the `Link` works as expected on this branch too, kindly fix this.